css/booklist.css

```diff
@@ -42,6 +44,13 @@
   white-space: normal;
   width: 250px;
 }
+.bottom_details{
+  justify-content: flex-end;
+  width: 180px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
```

Comment:
Please take a look at *Margin - Shorthand Property* and try to use it to write shorter and better code.
https://www.w3schools.com/css/css_margin.asp
This applies to every attribute that has `-left,-right,-top,-bottom`.